config/ssoe_idp_int_metadata_isam.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Not related to this PR, just for my own learning / posterity - why are we providing a hardcoded metada file when ruby-saml [generates one for us](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/saml_controller.rb)?

Reply:
This is the IDP metadata.